deployment/role.yaml

```diff
@@ -12,6 +12,18 @@ rules:
   verbs:
   - create
   - patch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Er delete, patch og update nødvendig her?

Reply:
Nei, ikke egentlig, men vi har ikke finjustert tilgang til noen ressurser for skiperator. Alle ressursene har bare copy-pasted full tilgang. Vi skulle egentlig gått over og gjort vurderinger per ressurs hva den egentlig trenger.